### PR TITLE
fix: beta fixes

### PR DESCRIPTION
**Pull request by 0xcadams**

### CODE REVIEW

File: components/OrchestratorList/index.tsx [hunk 1]
```diff
@@ -400,7 +400,7 @@ const OrchestratorList = ({ data, protocolData, pageSize = 10 }) => {
                       </Flex>
                       <Link
                         passHref
-                        href="https://github.com/livepeer/explorer/blob/main/ROI.md"
+                        href="https://docs.livepeer.org/delegators/how-to-guides/yield-calculation"
```
Comment (adamsoffer) [inline components/OrchestratorList/index.tsx:403]: Let's put this doc under a "Reference" category. The url should be `https://docs.livepeer.org/delegators/how-to-reference/yield-calculation`

Looks like existing category needs to be renamed from "Reference" to "Guides".  
image

**Review by adamsoffer — APPROVED** [review]
LGTM.

**Comment by 0xcadams** [comment]
Fixed!

**Comment by 0xcadams** [comment]
See here for docs updates https://github.com/livepeer/docs/pull/249/files

**Pull request merged**